cmds/minigames/connect4.js

```diff
@@ -105,55 +166,110 @@ class connect4 extends two_p_Game{
-                            redcollector.stop()
-                            yellowcollector.stop()
+                            await rmsg.edit(
+                                client.EmbedMaker(
+                                    msg,
+                                    game.stringify(),
+                                    client.colors.white
+                                )
+                            );
+                            await ymsg.edit(
+                                client.EmbedMaker(
+                                    msg,
+                                    game.stringify(),
+                                    client.colors / white
+                                )
+                            );
+                            iuser.send("Draw!");
+                            msg.author.send("Draw!");
+                            redcollector.stop();
+                            yellowcollector.stop();
                             break;
-                        }
-
-                })
-
-                yellowcollector.on("collect", async (reaction,user) =>{
-                
-                    if(reaction.emoji.name === "🛑"){
-
-                        rmsg.edit(client.EmbedMaker(msg,`Game abrupted by ${user.username}`, client.colors.black))
-                        ymsg.edit(client.EmbedMaker(msg,`Game abrupted by ${user.username}`, client.colors.black))
-                        end = true
-                        redcollector.stop()
-                        yellowcollector.stop()
-                        return
-
+                    }
+                });
+
+                yellowcollector.on("collect", async (reaction, user) => {
+                    if (reaction.emoji.name === "🛑") {
+                        rmsg.edit(
+                            client.EmbedMaker(
+                                msg,
+                                `Game abrupted by ${user.username}`,
+                                client.colors.black
+                            )
+                        );
+                        ymsg.edit(
+                            client.EmbedMaker(
+                                msg,
+                                `Game abrupted by ${user.username}`,
+                                client.colors.black
+                            )
+                        );
+                        end = true;
+                        redcollector.stop();
+                        yellowcollector.stop();
+                        return;
                     }
 
-                    console.log("y1")
-                    if(user.id === game.yellowid && current_color !== "yellow") return
-                    console.log("y2")
-
-                    switch(game.place("1️⃣2️⃣3️⃣4️⃣5️⃣6️⃣7️⃣🛑".indexOf(reaction.emoji.name)/3,current_color)){
-
+                    console.log("y1");
+                    if (user.id === game.yellowid && current_color !== "yellow")
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/xiaomiao-fong/Chen_bot/pullRequest?prid=7807535)
```suggestion
                    if (user.id === game.yellowid && current_color !== "yellow")
```